src/icon/checkCircleIcon.tsx

```diff
@@ -35,7 +35,7 @@ const StyledCheckCircleIcon = styled(CheckCircleIcon)`
     fill: ${props => (props.iconColor ? props.iconColor : color.green)};
   }
 
-  path:nth-child(2) {
+  path + path {
```

Comment:
sounds risky to have only the 2nd path on white.

Reply:
Should we use a classname in this case?